setup.py

```diff
@@ -45,7 +45,7 @@
       packages=['qp_klp'],
       scripts=['scripts/configure_klp', 'scripts/start_klp'],
       extras_require={'test': ["nose >= 0.10.1", "pep8"]},
-      install_requires=['click >= 3.3', 'future', 'pandas',
+      install_requires=['click >= 3.3', 'future', 'pandas < 2.0',
```

Comment:
What's the motivation for the pin?

Reply:
We were investigating why the build failed repeatedly while we were doing other things. At one point we thought this might be it and it was worth a random try.